hack/relprep.sh

```diff
@@ -80,29 +91,32 @@ for i in $crds; do
   fi
 done
 
-echo "Verifying that bundle-generated CSV (for testing) is equivalent to shipping CSV"
-ignorefields='createdAt|operators\.operatorframework\.io/builder|app\.kubernetes\.io/version'
-bcsv=bundle/manifests/keda.clusterserviceversion.yaml
-mcsv=config/manifests/bases/keda.clusterserviceversion.yaml
-if ! diff -u <(grep -vE "$ignorefields" < $bcsv) <(grep -vE "$ignorefields" < $mcsv) | \
-    sed 's#^--- /dev/fd/[0-9]*#--- '"$bcsv"'#;s#^+++ /dev/fd/[0-9]*#+++ '"$mcsv"'#'; then
-  echo "Error: It appears that there are non-trivial differences between $bcsv and $mcsv."
-  echo "As appropriate, make changes to $mcsv or the inputs to $bcsv, and re-run this script"
-  exit 1
-fi
-
 all_mods="$(go list -mod=readonly -m -f '{{ if and (not .Indirect) (not .Main)}}{{.Path}}{{end}}' all)"
 declare -A updated_mods
+to_update=()
+
+# force versions of k8s components
+for i in $kube_components; do
+    to_update+=("k8s.io/$i@$k8sver")
+    updated_mods["k8s.io/$i"]=1
+done
+
+# hack: force version of openshift API module based upon k8s->openshift version skew (e.g. 1.27 -> 4.14)
+openshift_branch="release-4.$(($(echo $k8sver | sed 's/v0\.\([0-9]*\)\.[0-9]*$/\1/')-13))"
```

Comment:
I've always wished there was a canonical file somewhere in github that had the openshift:kube version mapping, but I don't know of one either.  I think `-13`  (or `+2.87` ) is reasonable for now as long as it holds and we can get away with it -- I'd rather do that than maintain our own mapping table here and adding the new version every time we do the release.

Reply:
Okay, thanks for being a sounding board on this. I think I'm leaning in the same direction. Probably once it breaks, we'll notice when we try to build with an incompatibly-skewed version, and that can be our cue to fix the skew magic number.